eng/emsdk.proj

```diff
@@ -184,6 +184,10 @@
              Condition="'$(AssetManifestOS)' == '' or '$(AssetManifestOS)' == 'win' or '$(ForceBuildManifestOnly)' == 'true' or '$(ArcadeBuildFromSource)' == 'true'" 
              Targets="Build" 
              Properties="PreReleaseVersionLabel=$(PreReleaseVersionLabel);PreReleaseVersionIteration=$(PreReleaseVersionIteration)" />
+    <MSBuild Projects="$(MSBuildThisFileDirectory)nuget\Emsdk.Unversioned.Manifest\Emsdk.Unversioned.Manifest.pkgproj" 
+             Condition="'$(AssetManifestOS)' == '' or '$(AssetManifestOS)' == 'win' or '$(ForceBuildManifestOnly)' == 'true' or '$(ArcadeBuildFromSource)' == 'true'" 
+             Targets="Build" 
+             Properties="PreReleaseVersionLabel=$(PreReleaseVersionLabel);PreReleaseVersionIteration=$(PreReleaseVersionIteration)" />
```

Comment:
```suggestion
             Targets="Build" />
```

is probably enough but we could pick a better name and possibly include a .props with the full manifest name (or something)

Reply:
or and it should be transport not shipping